### PR TITLE
Pull request for markdown files

### PR DESCRIPTION
**Pull request by BascouFlorent**

### CODE REVIEW

File: biblio/references_all.bib [hunk 1]
```diff
@@ -11044,3 +11044,19 @@ @article{Vandel_Cassan_Lebre_Lecellier_Brehelin19
...
+
+
+@inproceedings{Rendle10,
+  author    = {Steffen Rendle},
+  title     = {Factorization Machines},
+  booktitle = {{ICDM} 2010, The 10th {IEEE} International Conference on Data Mining,
+               Sydney, Australia, 14-17 December 2010},
+  pages     = {995--1000},
+  year      = {2010},
+  crossref  = {DBLP:conf/icdm/2010},
```
Comment (josephsalmon) [inline biblio/references_all.bib]: PLease, can you remove the useless lines: crossref, url, timestamp, biburl, bibsource.

File: biblio/references_all.bib [hunk 2]
```diff
@@ -11044,3 +11044,19 @@ @article{Vandel_Cassan_Lebre_Lecellier_Brehelin19
 number = {1},
 page = {103}
 }
+
+
```
Comment (josephsalmon) [inline biblio/references_all.bib]: Sort also where it should be  (alphabetical order, before the Rendel12.

**Comment by josephsalmon** [comment]
I made two recommendations before accepting you PR on references_all. Please can you address them?

**Comment by BascouFlorent** [comment]
I think it's good

**Comment by josephsalmon** [comment]
thx looks good

**Pull request closed without merging**